### PR TITLE
Add SPID to ConnectionInfoSummary

### PR DESCRIPTION
**Pull request by smartguest**
Part of PR for ADS SPID display.

### CODE REVIEW

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]
This should wait until the other PRs are merged in before being updated, will review once those are completed.

**Review by Charles-Gagnon — APPROVED** [review]
Note that we don't need to publish a new version for this since it's just a typings update.

**Review by cheenamalhotra — APPROVED** [review]

**Pull request merged**
